### DISCUSSION TITLE
IBAX PROJESİ -TR-

### DISCUSSION THREAD

**Discussion by crosproata** [IBAX-io/go-ibax#193] [Ideas] [answered]
Genel olarak çok ince düşünülmüş bir proje,fee'ler olsun,gelecek planları olsun çok sağlam,TO THE MOON!!

**Comment by Grekny** [comment, marked as answer]
bende kesinlikle aynı düşünüyorum.

**Comment by Skips1** [comment]
Umarım Hayal kırıklığına uğramayız  :)
